modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/UserCard.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayCardWithUser} from '@clayui/card';
+import React, {useState} from 'react';
+
+const getDataAttributes = (data) => {
+	return data
+		? Object.entries(data).reduce((acc, [key, value]) => {
+				acc[`data-${key}`] = value;
+
+				return acc;
+		  }, {})
+		: {};
+};
+
+export default function UserCard({
+	actions = [],
+	componentId: _componentId,
+	cssClass,
+	inputName,
+	inputValue,
+	labels = [],
+	locale: _locale,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	selected: initialSelected,
+	selectable,
+	...otherProps
+}) {
+	const [selected, setSelected] = useState(initialSelected);
+
+	return (
+		<ClayCardWithUser
+			actions={actions?.map(({data, ...rest}) => {
+				const dataAttributes = getDataAttributes(data);
+
+				return {
```

Comment:
@carloslancha I was wondering if you didn't have any warning or errors about the `separator` property, because in my case I did - that's actually why I had to do [this](https://github.com/liferay-frontend/liferay-portal/pull/448/files#diff-5fd96603d12f2b582ee55300bf01605d4debe0ab7366ead15965e7baa49da709R53-R65)

Reply:
@carloslancha thanks for helping me solve this out - so basically (dropdown) items that aren't in group shouldn't be used with dividers/separators.